beluga/include/beluga/sensor.hpp

```diff
@@ -54,6 +54,7 @@
  *
  * \section SensorModelLinks See also
  * - beluga::LikelihoodFieldModel
+ * - beluga::Likelihood3DFieldModel
```

Comment:
```suggestion
 * - beluga::LikelihoodFieldModel3
```

Reply:
Changed in 2aabb60e1d02fcf2464ebca96f3cdb47be230d3a